Anamnesis/Actor/Posing/Visuals/SkeletonVisual3d.cs

```diff
@@ -585,18 +589,6 @@ public async Task SetActor(ActorMemory actor)
 			if (this.Actor.OffHand?.Model?.Skeleton != null)
 				this.AddBones(this.Actor.OffHand.Model.Skeleton, "oh_");
 
-			this.RaisePropertyChanged(nameof(SkeletonVisual3d.AllBones));
-			this.RaisePropertyChanged(nameof(SkeletonVisual3d.HairBones));
-			this.RaisePropertyChanged(nameof(SkeletonVisual3d.MetBones));
-			this.RaisePropertyChanged(nameof(SkeletonVisual3d.TopBones));
-			this.RaisePropertyChanged(nameof(SkeletonVisual3d.MainHandBones));
-			this.RaisePropertyChanged(nameof(SkeletonVisual3d.OffHandBones));
-			this.RaisePropertyChanged(nameof(SkeletonVisual3d.HasEquipmentBones));
-			this.RaisePropertyChanged(nameof(SkeletonVisual3d.HasWeaponBones));
-
-			if (!GposeService.Instance.IsGpose)
```

Comment:
This check is a duplicate. There's an identical one at the start of the method.

Reply:
Could be a relict of "Make sure no very unfortunate gpose leaving causes issues." Probably not but better to keep in mind it _was_ here once just in case.

Reply:
If you feel strongly about it, I can revert this change. However, the deleted gpose check is not very useful in my opinion. It is placed after the skeleton object is already populated with bones, the bones will just not be linked, which I would consier a malformed skeleton object.

Reply:
Nah its fine, likelyhood of it mattering is small but I just wanted to comment on it for the record, probably wouldnt have had you not said anything and just made a mental note lol.